app.py

```
@@ -0,0 +1,313 @@
+        "category": "Examples",
+        "status": "active",
+    },
+}
+
+
+@rt("/")
+def get():
+    TEXT_COLOR = "color: white;"
+    return Titled(
+        "",
+        Style("""
```

Ruff (0.14.1)

`Style` may be undefined, or defined from star imports

(F405)